best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
The best practice is `Use ES6`, I just point to the best solution (I know of) currently instead of keeping it ambiguous.